packages/app/src/app/pages/NewDashboard/types.ts

```diff
@@ -12,14 +13,20 @@ export type DashboardBaseFolder = {
 
 export type DashboardSandbox = {
   type: 'sandbox';
-  sandbox: SandboxFragmentDashboardFragment;
+  sandbox: SandboxFragmentDashboardFragment & {
```

Comment:
Maybe we should add the properties to the `SandboxFragmentDashboardFragment` instead? Since it seems like we're going to use it in other pages besides `Repositories` as well (from what I've seen).

Reply:
hm?

Reply:
I'm wondering, do we also show specific properties or menu items for repo sandboxes in other places than the repo page? If that's the case we could also change the fragment query of sandbox itself to include these properties.

Reply:
No we won't but that's the issue, if I try to get them TS will complain even thought I know they will exist since I am on the repos page